packages/teams-js/src/public/chat.ts

```diff
@@ -67,20 +74,26 @@ export namespace chat {
    * @beta
    */
   export function openChat(openChatRequest: OpenSingleChatRequest): Promise<void> {
+    const apiVersionTag = getApiVersionTag(chatTelemetryVersionNumber, ApiName.Chat_OpenChat);
```

Comment:
Does this need to be unique?  

If yes, from the function  "getApiVersionTag" itself, it doesnt seem that it will guarantee to return a unique string...
 return `${apiVersionNumber}_${functionName}`;

for example, two functions with same apiVersionNumber and same function names(function overload)

Reply:
taking a further look, since the functionName is a "controlled" name, which we define it in telemetry.ts, it may not be an issue if we update telemetry.ts file correctly for new updates, so I don't know if we ever run into the issue I called out above.
The comment is  just from purely coding perspective, if apiVersionTag needs to be unique, that function itself cannot guarantee that..

Reply:
That's a good thought. I think the biggest concern is what if a person add the same value to different keys and composed tags used the same versions. This could be a human mistake though very little chance would happen. I will take a look on it and to see if we could figure it out leveraging on ESLint check.